catalog.html

```diff
@@ -56,23 +56,201 @@
-            </details>
+            <form class="form-sidebar" action="#" method="get">
+                <details class="faq__item">
+                    <summary class="faq__question">Clothing</summary>
+                    <div class="faq__answer">
+                        <label class="form-checkbox">
+                            <input type="checkbox" class="input-hidden form-checkbox__input" id="combatant" name="combatant" required>
+                            <p class="form-checkbox__inner">
+                                <span class="form-checkbox__box"></span>
+                                <span class="form-checkbox__text">Combatant</span>
+                            </p>
+                        </label>
+                        <label class="form-checkbox">
+                            <input type="checkbox" class="input-hidden form-checkbox__input" id="ninja" name="ninja" required>
+                            <p class="form-checkbox__inner">
+                                <span class="form-checkbox__box"></span>
+                                <span class="form-checkbox__text">Ninja</span>
+                            </p>
+                        </label>
+                        <label class="form-checkbox">
+                            <input type="checkbox" class="input-hidden form-checkbox__input" id="name" name="name" required>
+                            <p class="form-checkbox__inner">
+                            <span class="form-checkbox__box"></span>
+                            <span class="form-checkbox__text">King</span>
+                            </p>
+                        </label>
+                        <label class="form-checkbox">
+                            <input type="checkbox" class="input-hidden form-checkbox__input" id="name" name="name" required>
+                            <p class="form-checkbox__inner">
+                            <span class="form-checkbox__box"></span>
+                            <span class="form-checkbox__text">Queen</span>
+                            </p>
+                        </label>
+                        <label class="form-checkbox">
+                            <input type="checkbox" class="input-hidden form-checkbox__input" id="name" name="name" required>
+                            <p class="form-checkbox__inner">
+                            <span class="form-checkbox__box"></span>
+                            <span class="form-checkbox__text">Robot</span>
+                            </p>
+                        </label>
+                    </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Eyes</summary>
+                    <div class="faq__answer">
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="yellow" required>
```

Comment:
Only one option can be chosen here. Add the equal name="yeas" (value="yellow") attribute for all radio buttons in the group.

<img width="359" alt="Screenshot 2024-12-16 at 01 08 59" src="https://github.com/user-attachments/assets/6a0cc758-db1d-4fce-b050-8e0004ec2dcf" />

Reply:
done